### ISSUE TITLE
Cropping an image

### ISSUE THREAD

**Issue by 100tomer** [uxbox/uxbox#2178]
**Is your feature request related to a problem? Please describe.**
Option to crop image after drag and drop it

**Describe the solution you'd like**
Option to crop the image like sketch

**Describe alternatives you've considered**
Currently it's possible to get to the same results but with adding nodes and moving them around

Thank you!

**Comment by myfunnyandy** [comment]
Hi, this is among our next priorities. We have plans to change the ways the images are managed at the viewport to enable capabilities like crop, image replacement and resizing options. Here's the story: https://tree.taiga.io/project/penpot/us/2983?milestone=262806 (I've added your request there).

**Comment by JPustkuchen** [comment]
Perhaps allowing to define a "transparent color" for images, might also be nice. Typically this is "White" for JPEG's, to be shown with transparency like PNG.

But I agree cropping might be the most important feature.

Thanks you soooo much for penpot! :)

**Comment by th0rgall** [comment]
This looks like a duplicate of #692, where there is a workaround with masks described here https://github.com/penpot/penpot/issues/692#issuecomment-785001118

**Comment by JPustkuchen** [comment]
@th0rgall indeed! The question is, if this is self-explaining and user-friendly enough? At least 13 people voting here didn't find that way and I guess there are more, who didn't take the time to search and vote this issue?

**Comment by astudentinearth** [comment]
@JPustkuchen Even though the workaround is fine, I'm pretty sure most users would expect a "Crop" functionality when they right click on the image. A lot of people won't think about using a mask, and some of them won't even know such functionality exists, especially if they are just starting out.

**Comment by JPustkuchen** [comment]
@astudentinearth yes, that's what I tried to imply in my comment :)

**Comment by th0rgall** [comment]
To be clear: I'd also be happier with built-in crop support, without workarounds! 🙂 The solution proposed in the Penpot Taiga looks promising.

I just mentioned the workaround here for those who were looking for a temporary solution to this problem (like me). I don't think this issue should be closed in favor of the duplicate.

@JPustkuchen It's also a little harder to find an old _closed_ issue, which I think explains why that issue didn't have many upvotes.

**Comment by myfunnyandy** [comment]
Update: the feature is already being developed in [our current sprint](https://tree.taiga.io/project/penpot/taskboard/sprint-49-75?order_by=status) :)

**Comment by elsatch** [comment]
Great to know! I was scratching my head for a while looking for a way to do it. I'll try the workaround with the mask for the moment.

**Comment by octogonz** [comment]
> Update: the feature is already being developed in [our current sprint](https://tree.taiga.io/project/penpot/taskboard/sprint-49-75?order_by=status) :)

Thank you!

**Comment by Mardoxx** [comment]
Fucking long sprint, guys

**Comment by fullheart** [comment]
It no longer seems to be in the sprint, it is currently in the backlog, see:
https://tree.taiga.io/project/penpot/us/4095

**Comment by Mardoxx** [comment]
Been playing with this and tbf I think masking is a much better mechanism than cropping (isn't it the same thing, after all?)

Perhaps crop context menu could just auto create a square mask?

--

Although for export as css/html  a /cropped/ image would be better (smaller file size) - if you want a static sized image
